### PR TITLE
fix(bayn): harden paper persistence boundaries

### PR DESCRIPTION
**Pull request by gregkonush**

### CODE REVIEW

File: services/bayn/src/broker/alpaca.ts [hunk 1]
```diff
@@ -666,6 +666,13 @@ const optionalTimestamp = (value: string | null): string | undefined => value ??
 const optionalMicros = (value: string | null, signed: boolean, name: string): string | undefined =>
   value === null ? undefined : decimalToMicros(value, signed, name)
 
+const positionMicros = (value: string, side: PositionSide, name: string): string => {
+  const decoded = BigInt(decimalToMicros(value, true, name))
+  if (decoded === 0n) throw new Error(`${name} must be non-zero`)
+  const magnitude = decoded < 0n ? -decoded : decoded
+  return (side === PositionSide.Short ? -magnitude : magnitude).toString()
```
[the remark this answers is not in the document]
Comment (gregkonush) [reply services/bayn/src/broker/alpaca.ts]: Fixed in 7516736e9e. positionMicros now validates the side-normalized value against the signed i128 range, so applying long to an I128_MIN magnitude fails at the adapter boundary. The regression feeds that exact edge case and verifies INVALID_RESPONSE. Current-main validation: 206 tests, 0 failures; tsc, formatting, both Oxlint modes, and build pass.
